### PR TITLE
Setup cocoapod

### PR DESCRIPTION
**Pull request by felixLam**

### CODE REVIEW

File: ISHPullUp.podspec [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+Pod::Spec.new do |s|
+  s.name             = 'ISHPullUp'
+  s.version          = '0.9.0'
+  s.summary          = 'Vertical split viewcontroller with pull up gesture as seen in the iOS 10 Maps app '
```
Comment (shagedorn) [inline ISHPullUp.podspec]: split view controller?

Comment (felixLam) [inline ISHPullUp.podspec]: Fixed

File: ISHPullUp.podspec [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ISHPullUp provides a simple UIViewControlller subclass with two child controllers. The layout can be managed entirely via delegation and is easy to use with autolayout.
+
+Two view subclasses are provided to make beautiful iOS10 style designs easier. ISHPullUpHandleView provides a drag handle as seen in the notification center or Maps app with three states: up, neutral, down. ISHPullUpRoundedView provides the perfect backing view for your bottom view controller with a hairline border and rounded top corners.
+                       DESC
+
+  s.homepage         = 'https://github.com/iosphere/ISHPullUp'
+  s.screenshots     = 'https://github.com/iosphere/ISHPullUp/raw/master/screenshot.jpg'
+  s.license          = { :type => 'MIT', :file => 'LICENSE' }
+  s.author           = { 'Felix Lamouroux' => 'felix@iosphere.de' }
+  s.source           = { :git => 'https://github.com/<GITHUB_USERNAME>/ISHPullUp2.git', :tag => s.version.to_s }
```
Comment (shagedorn) [inline ISHPullUp.podspec]: `https://github.com/iosphere/ISHPullUp.git`?

File: ISHPullUp.podspec (hunk 1 above)
Comment (mollidor) [inline ISHPullUp.podspec]: yip. From `UISplitViewController.h`

> If the view controller has a split view controller as its ancestor, return it.

File: ISHPullUp.podspec [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  s.social_media_url = 'https://twitter.com/iosphere'
+
+  s.ios.deployment_target = '8.0'
+
+  s.source_files = 'ISHPullUp/*'
```
Comment (shagedorn) [inline ISHPullUp.podspec]: `.{h,m}`?

File: ISHPullUp.podspec [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  s.version          = '0.9.0'
+  s.summary          = 'Vertical split view controller with pull up gesture as seen in the iOS 10 Maps app '
+  s.description      = <<-DESC
+ISHPullUp provides a simple UIViewControlller subclass with two child controllers. The layout can be managed entirely via delegation and is easy to use with autolayout.
+
+Two view subclasses are provided to make beautiful iOS10 style designs easier. ISHPullUpHandleView provides a drag handle as seen in the notification center or Maps app with three states: up, neutral, down. ISHPullUpRoundedView provides the perfect backing view for your bottom view controller with a hairline border and rounded top corners.
+                       DESC
+
+  s.homepage         = 'https://github.com/iosphere/ISHPullUp'
+  s.screenshots     = 'https://github.com/iosphere/ISHPullUp/raw/master/screenshot.jpg'
```
Comment (mollidor) [inline ISHPullUp.podspec]: indentation

File: ISHPullUp.podspec [hunk 5]
```diff
@@ -0,0 +1,23 @@
+
+Pod::Spec.new do |s|
+  s.name             = 'ISHPullUp'
+  s.version          = '0.9.0'
+  s.summary          = 'Vertical split view controller with pull up gesture as seen in the iOS 10 Maps app '
```
Comment (mollidor) [inline ISHPullUp.podspec]: trailing whitespace

File: ISHPullUp.podspec (hunk 3 above)
Comment (felixLam) [inline ISHPullUp.podspec]: Fixed and tested agaib

File: ISHPullUp.podspec (hunk 5 above)
Comment (felixLam) [inline ISHPullUp.podspec]: fixed

**Comment by shagedorn** [comment]
Please add setup instructions to the README.

After merging, you need to:

* set & push the `0.9` tag
* run `pod trunk lint`
* if that succeeds, `pod trunk push`

You probably need to claim the repo when running `pod trunk`. I you want me to manage updates in the future, please add me, too.

**Pull request merged**